scripts/copy_files.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const path = require("path");
+
+// Define the source and destination file paths
+const sourceFile = path.join(__dirname, '../src/external/aria');
```

Comment:
I'd suggest to not use any OS platform specific path like "../a/b/c". 

You can use `path.join(path.dirname(__dirname), "src", "external", "aria")`